src/content/docs/en/guides/internationalization.mdx

```diff
@@ -0,0 +1,259 @@
+
+getAbsoluteLocaleUrl("fr", ""); // returns https://example.com/fr
+getAbsoluteLocaleUrl("fr", "getting-started"); // returns https://example.com/fr/getting-started
+getAbsoluteLocaleUrl("fr_CA", "getting-started", {
+  prependWith: "blog"
+}); // returns https://example.com/blog/fr-ca/getting-started
+
+getAbsoluteLocaleUrl("fr_CA", "getting-started", {
+  prependWith: "blog",
+  normalizeLocale: false
+}); // returns https://example.com/blog/fr_CA/getting-started
+
+---
+```
+
+### `getRelativeLocaleUrlList()`
+
+It works like [`getRelativeLocaleUrl`](#getrelativelocaleurl), so it returns a list of relative paths for all the locales.
+
+
+`getRelativeLocaleUrlList(locale: string, options?: GetLocaleOptions): string[]`
+
+### `getAbsoluteLocaleUrlList()`
+
+`getAbsoluteLocaleUrlList(locale: string, options?: GetLocaleOptions): string[]`
+
+
+It works like [`getAbsoluteLocaleUrl`](#getabsolutelocaleurl), so it returns a list of relative paths for all the locales.
+
+
+## Routing strategy 
+
+The option `i18n.routingStrategy` allows to change the behaviour of the Astro routes
+
+### `'prefix-other-locales'`
+
+```js title="astro.config.mjs" ins={7}
+import { defineConfig } from "astro/config"
+export default defineConfig({
+  experimental: {
+    i18n: {
+      defaultLocale: "en",
+      locales: ["es", "en", "fr"],
+      routingStrategy: "prefix-other-locales"
+    }
+  }
+})
+```
+
+This is the **default** value. With this option all the routes *without* locales belong to the default locale (`i18n.defaultLocale`), and the rest of the locales must have their locale code (prefix) in the URL.
+
+- You want to have `example.com/content` for the default locale.
+- You want to have `example.com/es/content` for the other locales.
+- You want to have `example.com/` to redirect (302 status code) to `example.come/<i18n.defaultLocale>`.
+
+### `'prefix-always'`
+
+```js title="astro.config.mjs" ins={7}
+import { defineConfig } from "astro/config"
+export default defineConfig({
+  experimental: {
+    i18n: {
+      defaultLocale: "en",
+      locales: ["es", "en", "fr"],
+      routingStrategy: "prefix-always"
+    }
+  }
+})
+```
+
+With this option, all the routes must have their locale code (prefix) in URL.
+
+- You want to have `example.com/[locale]/content` for all routes.
+- You want `example.com/content` to return a 404 (not found) status code.
+
+
+## Browser language detection
+
+When i18n routing is enabled, two new properties are available via [`Astro` global object](/en/reference/api-reference/#astro-global). These values are **only available in SSR**.
+
+The values are calculated by deducing the languages supported by the browser - via the [`Accept-Language`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language) header -  and the locales supported by the website using [`i18n.locales`].
+
+- `Astro.preferredLocale`: it's the locale requested by the browser and supported by the website. The value is `undefined` if:
```

Comment:
I'm assuming this is the first match between the user's list and the website's list.

```suggestion
- `Astro.preferredLocale` is the first locale requested by the browser and supported by the website. The value is `undefined` if:
```